includes/Classifai/Providers/OpenAI/Embeddings.php

```diff
@@ -235,6 +384,55 @@ public function get_threshold( string $taxonomy = '' ): float {
 		return apply_filters( 'classifai_threshold', $threshold, $taxonomy );
 	}
 
+	/**
+	 * Regenerate embeddings.
+	 *
+	 * This will regenerate embeddings for all terms
+	 * and delete existing post embeddings. Useful to run
+	 * anytime the model or dimensions are changed.
+	 */
+	public function regenerate_embeddings() {
+		$feature  = new Classification();
+		$settings = $feature->get_settings();
+
+		if (
+			! $feature->is_feature_enabled() ||
+			$feature->get_feature_provider_instance()::ID !== static::ID
+		) {
+			return;
+		}
+
+		// Regenerate embeddings for all terms.
+		foreach ( array_keys( $this->nlu_features ) as $feature_name ) {
+			if ( isset( $settings[ $feature_name ] ) && 1 === (int) $settings[ $feature_name ] ) {
+				// TODO: this also needs fixed for the same reasons as the other trigger_taxonomy_update call.
+				$this->trigger_taxonomy_update( $feature_name, false );
```

Comment:
```suggestion
				$this->trigger_taxonomy_update( $feature_name, true );
```

Maybe we want to pass `true` here in second argument? as passing false not generating embeddings for me.